moveit_core/collision_detection/include/moveit/collision_detection/world.h

```diff
@@ -105,13 +106,19 @@ class World
      *
      * @copydetails shapes_ */
     EigenSTL::vector_Isometry3d shape_poses_;
+
+    /** \brief Transforms to subframes on the object. Transforms are applied to the link.
+     *  Use these to define points of interest on the object to plan with
+     *  (e.g. screwdriver_tip, kettle_spout, mug_base).
+     * */
+    std::map<std::string, Eigen::Isometry3d> subframe_poses_;
```

Comment:
You need to use an aligned allocator for the Eigen transforms!
What about directly using [`moveit::core::Transforms`](https://github.com/ros-planning/moveit/blob/7fdc7f35f763c5e6784b5a72665188b2426fc009/moveit_core/transforms/include/moveit/transforms/transforms.h#L61) or at least [`moveit::core::FixedTransformsMap`](https://github.com/ros-planning/moveit/blob/7fdc7f35f763c5e6784b5a72665188b2426fc009/moveit_core/transforms/include/moveit/transforms/transforms.h#L54)?

Reply:
I don't know what an aligned allocator is, but I can replace the `std::map` with `moveit::core::FixedTransformsMap` if that works?